backend/src/models/account.model.ts

```diff
@@ -0,0 +1,16 @@
+interface Account {
+    user: {
+        id?: number;
+        name?: string,
+        email: string,
+        password?: string,
+        isActive?: Boolean,
+        createdAt?: Date,
+        updatedAt?: Date,
+        deletedAt?: Date,
+    }
+    token?: string,
```

Comment:
The token shouldn't be here because it is not stored in the database. The Account entity should reflect the database table as much as possible.

Reply:
remember that all models have UpperCamelCase style, as all of them are public. 

```suggestion
   interface Account {
        ID?: number;
        Name: string,
        Email: string,
        Hash: string,
        isActive: boolean,
        CreatedAt?: Date,
        UpdatedAt?: Date,
        DeletedAt?: Date,
   }
```

Reply:
Done. Thank you for the tip.